### PR TITLE
Create OpenShift route manifest file

### LINKED ISSUE #24: Create route object yaml
We are wiring our web-server service to the world via OpenShift Route object. This, however, is setup via OS UI... we should create the route yaml to  be able to export it with the whole package. An alternative would be setup the k8s ingress object.

### PR DESCRIPTION
**Pull request by damianpm**
Closes https://github.com/3scale/k8sapp-initiative/issues/24

Creates OpenShift Route yaml file

**Verification:**
`oc apply -f web-server_route.yml`

Route already created: http://k8sinitiative.apps.dev-eng-ocp4-5.dev.3sca.net/

### CODE REVIEW

File: web-server_route.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: web-server
+  namespace: k8sinitiative
+  resourceVersion: "72299960"
+  selfLink: /apis/route.openshift.io/v1/namespaces/k8sinitiative/routes/web-server
+  uid: 18489824-19e1-4965-8cbb-f2f3cf2d05b4
```
Comment (didierofrivia) [inline web-server_route.yaml]: I don't think these attributes are necessary, are them? how did you generate them?

File: web-server_route.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+spec:
+  host: k8sinitiative.apps.dev-eng-ocp4-5.dev.3sca.net
+  port:
+    targetPort: 8080
+  to:
+    kind: Service
+    name: web-server
+    weight: 100
+  wildcardPolicy: None
```
Comment (didierofrivia) [inline web-server_route.yaml]: This one too, default value?

File: web-server_route.yaml (hunk 1 above)
Comment (damianpm) [reply web-server_route.yaml]: These 3 items: `resourceVersion`, `selfLink` and `uid` are  automatically added when you run `oc apply`
I was unsure if commit it or not, I couldn't find anything useful in the docs, but making a [quick search on GH](https://github.com/search?q=selfLink%3A+%2Fapis%2Froute.openshift.io%2Fv1%2F&type=code) I've found out that people is actually committing

Comment (damianpm) [reply web-server_route.yaml]: But, deleting them and creating the route again is working fine. 
I think we can delete them

File: web-server_route.yaml (hunk 2 above)
Comment (damianpm) [reply web-server_route.yaml]: I copied from an example, but it seems is not needed

**Review by didierofrivia — APPROVED** [review]

**Pull request merged**